src/app/components/audio-recordings/pages/download/download.component.ts

```diff
@@ -0,0 +1,389 @@
+import { Component, OnInit } from "@angular/core";
+import { ActivatedRoute } from "@angular/router";
+import { AudioRecordingsService } from "@baw-api/audio-recording/audio-recordings.service";
+import { Filters } from "@baw-api/baw-api.service";
+import { projectResolvers } from "@baw-api/project/projects.service";
+import { regionResolvers } from "@baw-api/region/regions.service";
+import {
+  hasResolvedSuccessfully,
+  ResolvedModelList,
+  retrieveResolvers,
+} from "@baw-api/resolver-common";
+import { siteResolvers } from "@baw-api/site/sites.service";
+import { contactUsMenuItem } from "@components/about/about.menus";
+import {
+  audioRecordingMenuItems,
+  audioRecordingsCategory,
+} from "@components/audio-recordings/audio-recording.menus";
+import { myAccountMenuItem } from "@components/profile/profile.menus";
+import { PageComponent } from "@helpers/page/pageComponent";
+import { IPageInfo, PageInfo } from "@helpers/page/pageInfo";
+import { withUnsubscribe } from "@helpers/unsubscribe/unsubscribe";
+import { Id } from "@interfaces/apiInterfaces";
+import { AudioRecording } from "@models/AudioRecording";
+import { Project } from "@models/Project";
+import { Region } from "@models/Region";
+import { Site } from "@models/Site";
+import { FormlyFieldConfig } from "@ngx-formly/core";
+import { ConfigService } from "@services/config/config.service";
+import { DateTime } from "luxon";
+
+const projectKey = "project";
+const regionKey = "region";
+const siteKey = "site";
+
+interface Model {
+  projectId?: Id;
+  regionId?: Id;
+  siteId?: Id;
```

Comment:
This model is just the output of the form, without making the actual form inputs which would output arrays, i cant have this as an array. I can however convert these values to arrays when being passed to the service so that things are prepared for the future change

Reply:
I mean, isn't the formly spec for these controls a `select`? so by adding the `multiple` attribute this would happen anyway?